view/theme/frio/templates/threaded_conversation.tpl

```diff
@@ -20,6 +20,5 @@
 <button type="button" id="item-delete-selected" class="btn btn-link" title="{{$dropping}}" onclick="deleteCheckedItems();" data-toggle="tooltip">
 	<i class="fa fa-trash" aria-hidden="true"></i>
 </button>
-<img id="item-delete-selected-rotator" class="like-rotator" src="images/rotator.gif" style="display: none;" />
```

Comment:
That image is used in other themes as well. Has it to e removed from there as well?

Reply:
Nope, the feature is only changed on the frio theme. Vier still uses it but it doesn't bother me over there because I'm not using the theme.